jobs/v3/howto/sample_test.go

```diff
@@ -8,11 +8,25 @@ import (
 	"bytes"
 	"strings"
 	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
```

Comment:
@tbpg , With this internal testutil, when our QA checking out code from master of GoogleCloudPlatform to run test,
we met this error:
```sample_test.go:12:2: use of internal package github.com/GoogleCloudPlatform/golang-samples/internal/testutil not allowed
FAIL _/usr/local/google/home/myao/CjdCodeSamples/go/golang-samples/jobs/v3/howto [setup failed]```
Could you share how to bypass this in our QA env?
Thanks,
Tanya

Reply:
Looks like the code wasn't properly checked out. You should run:

```
go get -t -v github.com/GoogleCloudPlatform/golang-samples/jobs/...
```

You should use `go get` to make sure the package ends up in the right location relative to `GOPATH`. In particular, this package should be at `$GOPATH/src/github.com/GoogleCloudPlatform/golang-samples`.

The `-t` flag indicates get the dependencies required to run the tests.